src/api/Alert/index.ts

```diff
@@ -189,6 +190,18 @@ export class AlertAPI extends Request {
     });
   }
 
+  /**
+   * Assign one or more cases to a user.
+   */
+  public assignCases(params: IAssignCasesParams) {
```

Comment:
I think we are missing the return type for this function here

Reply:
@yigitakbulut Not sure who decided this but no function in the rest-client has a return type you can check

Reply:
I think the pattern is something like this:
`
public assignCases(this: Request<void>, params: IAssignCaseParams)
`

However since we return void, I agree that this can be discarded

Reply:
Indeed 👍